receiver/fluentforwardreceiver/metadata.yaml

```diff
@@ -7,3 +7,5 @@ status:
   distributions: [contrib, splunk, observiq, sumo]
   codeowners:
     active: [dmitryax]
+tests:
+  config:
```

Comment:
Not sure if this is just a rendering issue, or a spacing issue, but this looks different than in other PRs:

<img width="356" alt="Screenshot 2023-12-17 at 7 50 00 AM" src="https://github.com/open-telemetry/opentelemetry-collector-contrib/assets/223565/3f3137af-f349-4473-a50f-4142b8c26011">

Reply:
Weird apparently this is caused by there not being a newline after the `active`